Project/Source/GameplaySystems.cpp

```diff
@@ -235,7 +267,6 @@ float Screen::GetScreenWitdh() {
 }
 
 float Screen::GetScreenHeight() {
-
 	return static_cast<float>(App->window->GetHeight());
 }
 
```

Comment:
Warning: conversion ... Possible loss of data in.
float2 Screen::GetResolution() {
	return float2(App->window->GetWidth(), App->window->GetHeight());
}

Add explicit (float) conversion.

(It is not an issue from this PR, sorry, but we wont need to open a new branch if you change it now :D  )

Reply:
Done